### PR TITLE
Cloud docs restructure

### PR DESCRIPTION
**Pull request by sophia-wiley**
Restructured the Cloud docs in the sidebar. Broke up the Managing Airbyte Cloud doc into separate docs. Updated links.

### CODE REVIEW

File: docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Manage Airbyte Cloud notifications
+
+This page provides guidance on how to manage notifications for Airbyte Cloud, allowing you to stay up-to-date on the activities in your workspace. 
+
+## Set up Slack notifications
```
Comment (sophia-wiley) [inline docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md:5]: Should I rename this to "Set up Webhook notifications"?

File: docs/cloud/managing-airbyte-cloud/dbt-cloud-integration.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# Using the dbt Cloud integration 
+# Utilize the dbt Cloud integration
```
Comment (sophia-wiley) [inline docs/cloud/managing-airbyte-cloud/dbt-cloud-integration.md:1]: Should I change "Utilize" to "Use" or another verb?

Comment (Amruta-Ranade) [reply docs/cloud/managing-airbyte-cloud/dbt-cloud-integration.md:1]: yep, utilize > use

File: docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md (hunk 1 above)
Comment (Amruta-Ranade) [reply docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md:5]: Does this workflow for products other than Slack? If yes, then changing the title makes sense. If not, the current title is better

**Review by Amruta-Ranade — CHANGES_REQUESTED** [review]
Great work! One thing we need to check for is if we need to add redirects in the /docusaurus/docusaurus.config.js file.

**Review by Amruta-Ranade — APPROVED** [review]
Great job!

**Pull request merged**